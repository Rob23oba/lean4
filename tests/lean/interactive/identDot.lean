macro "#test " i:identDot : command => `(#print $i)

namespace MyNamespace

def foo : Nat := 33

end MyNamespace

#test MyNamespace.
                --^ textDocument/completion
