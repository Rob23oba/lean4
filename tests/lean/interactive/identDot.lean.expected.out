{"textDocument": {"uri": "file:///identDot.lean"},
 "position": {"line": 8, "character": 18}}
{"items":
 [{"label": "foo",
   "kind": 21,
   "data":
   {"params":
    {"textDocument": {"uri": "file:///identDot.lean"},
     "position": {"line": 8, "character": 18}},
    "id": {"const": {"declName": "MyNamespace.foo"}},
    "cPos": 0}}],
 "isIncomplete": false}
